portal/v2/src/ClusterDetailList.tsx

```diff
@@ -54,6 +54,60 @@ const clusterDetailHeadings: IClusterDetails = {
   installStatus: "Installation Status",
 }
 
+export interface FeatureFlags {
+  "aro.alertwebhook.enabled" : string
+  "aro.autosizednodes.enable": string
+  "aro.azuresubnets.enabled": string
+  "aro.azuresubnets.nsg.managed": string
+  "aro.azuresubnets.serviceendpoint.managed": string
+  "aro.banner.enabled": string
+  "aro.checker.enabled": string
+  "aro.dnsmasq.enabled": string
+  "aro.genevalogging.enabled": string
+  "aro.imageconfig.enabled": string
+  "aro.machine.enabled": string
+  "aro.machinehealthcheck.enabled": string
+  "aro.machinehealthcheck.managed": string
+  "aro.machineset.enabled": string
+  "aro.monitoring.enabled": string
+  "aro.nodedrainer.enabled": string
+  "aro.pullsecret.enabled": string
+  "aro.pullsecret.managed": string
+  "aro.rbac.enabled": string
+  "aro.routefix.enabled": string
+  "aro.storageaccounts.enabled": string
+  "aro.workaround.enabled": string
+  "rh.srep.muo.enabled": string
+  "rh.srep.muo.managed": string
+}
+
+const featureFlagsHeadings: FeatureFlags = {
+  "aro.alertwebhook.enabled": "ARO Alertwebhook Enabled",
+  "aro.autosizednodes.enable": "ARO Autosizednodes Enable",
+  "aro.azuresubnets.enabled": "ARO Azuresubnets Enabled",
+  "aro.azuresubnets.nsg.managed": "ARO Azuresubnets NSG Managed",
+  "aro.azuresubnets.serviceendpoint.managed": "ARO Azuresubnets Serviceendpoint Managed",
+  "aro.banner.enabled": "ARO Banner Enabled",
+  "aro.checker.enabled": "ARO Checker Enabled",
+  "aro.dnsmasq.enabled": "ARO Dnsmasq Enabled",
+  "aro.genevalogging.enabled": "ARO Genevalogging Enabled",
+  "aro.imageconfig.enabled": "ARO Imageconfig Enabled",
+  "aro.machine.enabled": "ARO Machine Enabled",
+  "aro.machinehealthcheck.enabled": "ARO Machinehealthcheck Enabled",
+  "aro.machinehealthcheck.managed": "ARO Machinehealthcheck Managed",
+  "aro.machineset.enabled": "ARO Machineset Enabled",
+  "aro.monitoring.enabled": "ARO Monitoring Enabled",
+  "aro.nodedrainer.enabled": "ARO Nodedrainer Enabled",
+  "aro.pullsecret.enabled": "ARO Pullsecret Enabled",
+  "aro.pullsecret.managed": "ARO Pullsecret Managed",
+  "aro.rbac.enabled": "ARO RBAC Enabled",
+  "aro.routefix.enabled": "ARO Routefix Enabled",
+  "aro.storageaccounts.enabled": "ARO Storageaccounts Enabled",
+  "aro.workaround.enabled": "ARO Workaround Enabled",
+  "rh.srep.muo.enabled": "RH SREP MUO Enabled",
+  "rh.srep.muo.managed": "RH SREP MUO Managed",
```

Comment:
I'm on the fence about this change. Perhaps we can update where we configure the flags in the RP codebase to point devs over here to make sure this also gets updated. Otherwise new/changed flags won't appear in the portal.

Reply:
Added comments to `defaults.go` and `ClusterDetailList.tsx` for pointing devs to get the FeatureFlags interface updated here.